lib/puppet/module/task.rb

```diff
@@ -0,0 +1,85 @@
+require 'puppet/util/logging'
```

Comment:
It doesn't look like we're using any of the logging facilities directly in this class. Do we need to require this? Maybe we could just do `require 'puppet/error'` instead since we subclass it later.